lib/src/main/java/com/futurewei/alcor/common/db/ignite/MockIgniteServer.java

```diff
@@ -18,25 +18,35 @@
 import org.apache.ignite.Ignite;
 import org.apache.ignite.IgniteException;
 import org.apache.ignite.Ignition;
+import org.apache.ignite.configuration.ClientConnectorConfiguration;
```

Comment:
@chenpiaoping I think the MockIgniteServer and MockRedisServer are very good change which we want to merge to Master quick as other services could start using that. 

Is it possible that we split this PR into two by creating a new PR solely on the Mock change, which is supposed to consist of just a few files? Let me know.

Reply:
If there is no problem with this PR so far, I think we can merge it into the master,  I'll come back and add the part of interaction with route manager and dataplane later.

Reply:
It's a bit troublesome to split MockIgniteServer from this PR, because many of the UTs of ip manager are based on MockIgniteServer.

Reply:
@chenpiaoping This PR is kind of huge with over 100 files change.  Some files are still under review.

Let us check in small change first.

Reply:
@chenpiaoping I miss your last comment about this PR has dependency on the MockService. In this case, it is okay to keep MockService here. Let me try to review the rest and merge soon then.

Reply:
Thanks, xieus.